toolkit/resources/manifests/package/license_file_exceptions.json

```diff
@@ -0,0 +1,37 @@
+{
+    "_comment1": "This file is used to allow specific files to be omitted from the license check process.",
+    "_comment2": "Each PkgException entry is a {'PackageName' and 'IgnoredFilesRegexList'}",
+    "_comment3": "The 'PackageName' is the name of the (sub)package to which the exception applies.",
+    "_comment4": "The 'IgnoredFilesRegexList' is a list of regular expressions that match files to be omitted from the license check.",
+    "_comment5": "The 'GlobalExceptionsRegexList' is a list of regular expressions that match all packages.",
+
+    "PkgExceptions": [
+        {
+            "PackageName": "gcc",
+            "IgnoredFilesRegexList": [
+                "^/usr/share/man/man7/gpl.7.gz$"
+            ]
+        },
+        {
+            "PackageName": "libdb-docs",
+            "IgnoredFilesRegexList": [
+                "^/usr/share/doc/libdb-5.3.28/installation/build_unix_freebsd.html$",
+                "^/usr/share/doc/libdb-5.3.28/license/license_db.html$"
```

Comment:
Do we want to be tied to the 5.3.28 version?

Reply:
I was initially going to to explicit, but thinking back I agree that is probably going to be an ongoing pain to stay on top of. I'll replace with `[0-9\\.]+` which should match the version numbers.

Reply:
Also realized that the `.` in the paths should be `\\.`.